### PR TITLE
Migrations: PRONOM89

### PR DESCRIPTION
**Pull request by Hwesta**
Update for PRONOM89.

After new formats are recategorized, this should be updated with the changes.

### CODE REVIEW

**Comment by sevein** [comment]
Created with artefactual/archivematica-devtools#25

**Comment by nickwilkinson** [comment]
[AgentArchives > merge Bentley PRs](https://trello.com/c/pTkoVwZg/17-agentarchives-merge-bentley-prs)

**Comment by nickwilkinson** [comment]
[Problem: FITS script is outdated, refs #8647 (fpr admin changes)](https://trello.com/c/pIr1Ty2G/22-problem-fits-script-is-outdated-refs-8647-fpr-admin-changes)

File: fpr/migrations/0007_pronom_89.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from fpr.models import Format, FormatVersion, IDRule
```
Comment (sevein) [inline fpr/migrations/0007_pronom_89.py]: We can't do this. I've found the only way to deal with this issue properly is updating django-autoslug to 1.9.3.

**Comment by sevein** [comment]
See #51.

**Pull request closed without merging**